gitensite/apps/bookinfo/models.py

```diff
@@ -73,4 +77,16 @@ def metadata(self):
         if not self._pandata:
             self._pandata=Pandata()
             self._pandata.load(self.yaml)
-        return self._pandata.metadata
\ No newline at end of file
+        return self._pandata.metadata
+
+class Cover(models.Model):
+    cover_id = models.IntegerField(unique=True)
```

Comment:
wait.. is this duplicating the primary key?

Reply:
I didn't realize that Django automatically creates a primary key column. Thanks for bringing this up, we will get rid of the `cover_id` column